### PR TITLE
5.1.1_r28 --> 5.1.1_r36; Sync device trees; and remove empty repository

### PR DESCRIPTION
**Pull request by Startrek852**

### CODE REVIEW

File: default.xml [hunk 1]
```diff
@@ -40,24 +40,24 @@
   <!--project path="developers/docs" name="platform/developers/docs" /> -->
   <!--project path="developers/samples/android" name="platform/developers/samples/android" /> -->
   <!--project path="development" name="android_development" remote="omnirom" revision="android-5.1" groups="pdk-cw-fs" /> -->
```
Comment (lj50036) [inline default.xml]: Are these needed for building a device ?

File: default.xml [hunk 2]
```diff
@@ -461,7 +461,6 @@
   <project path="prebuilts/clang/linux-x86/host/3.4" name="platform/prebuilts/clang/linux-x86/host/3.4" groups="pdk,linux" />
   <project path="prebuilts/clang/linux-x86/host/3.5" name="platform/prebuilts/clang/linux-x86/host/3.5" groups="pdk,linux" />
   <project path="prebuilts/clang/linux-x86/mips/3.3" name="platform/prebuilts/clang/linux-x86/mips/3.3" groups="linux,mips,pdk-cw-fs" />
-  <project path="prebuilts/clang/linux-x86/x86/3.3" name="platform/prebuilts/clang/linux-x86/x86/3.3" groups="linux,pdk-cw-fs,x86" />
```
Comment (lj50036) [inline default.xml]: I have no issue commenting this out, but not removing it.

File: default.xml [hunk 3]
```diff
@@ -19,7 +19,7 @@
            review="gerrit.twrp.me"
            fetch="https://github.com/TeamWin" />
 
-  <default revision="refs/tags/android-5.1.1_r28"
+  <default revision="refs/tags/android-5.1.1_r36"
```
Comment (lj50036) [inline default.xml]: How can I update beyond OMNI ?

Comment (Startrek852) [inline default.xml]: I'm sorry, I don't understand.

File: default.xml (hunk 2 above)
Comment (Startrek852) [inline default.xml]: OK, I'll comment it out.

File: default.xml (hunk 1 above)
Comment (Startrek852) [inline default.xml]: They are required for building generic devices.

File: default.xml (hunk 2 above)
Comment (Startrek852) [inline default.xml]: I commented it out.

File: default.xml (hunk 1 above)
Comment (Startrek852) [inline default.xml]: Sorry, I didn't read that right. I don't think those 3 lines are important for building a device or for TWRP.

**Comment by lj50036** [comment]
@Startrek852, Can you do a pull request for every change ?

**Comment by Startrek852** [comment]
Sure.

**Comment by Startrek852** [comment]
OK, I made 3 different pull requests for every commit. Going to close this one now.

**Pull request closed without merging**